src/main/java/core/basesyntax/service/impl/ReportCreatorServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportCreatorService;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportCreatorServiceImpl implements ReportCreatorService {
+    private static final String HEADER = "fruit,quantity";

```

Comment:
same, it is better to path HEADER through constructor, so we can use this class to create report with different header